packages/cmrjs/tests/cmr-utils/test-cmr-utils.js

```diff
@@ -187,7 +187,7 @@ test.serial('updateEcho10XMLMetadata adds granule files correctly to OnlineAcces
   ];
   const AssociatedBrowseExpected = [
     {
-      URL: 'https://cumulus-test-sandbox-public.s3.amazonaws.com/MOD09GQ___006/TESTFIXTUREDIR/MOD09GQ.A6391489.a3Odk1.006.3900731509248_ndvi.jpg',
+      URL: `${distEndpoint}/cumulus-test-sandbox-public/MOD09GQ___006/TESTFIXTUREDIR/MOD09GQ.A6391489.a3Odk1.006.3900731509248_ndvi.jpg`,
```

Comment:
Not important, but how/where does this get replaced with the real value?

Reply:
It's just string template and distEndpoint is defined above. https://github.com/nasa/cumulus/pull/932/files#diff-491e3ac32e9806e9fe4ca7abfed466b7R158